src/code_8092334.c

```diff
@@ -1,6 +1,7 @@
 #include "global.h"
 #include "code_8092334.h"
 #include "memory.h"
+#include "constants/item.h"
```

Comment:
Probably can just include the constants file in `item.h` similar to pokeemerald.

Reply:
I like that, done.